packages/shared/comms/context.ts

```diff
@@ -0,0 +1,445 @@
+import { commConfigurations, parcelLimits } from 'config'
```

Comment:
I'm assuming most of this code was moved without changes, other than converting some global functions into methods of comms context.

If there's something more significant than that that I should review, please let me know!